### ISSUE TITLE
Failed ad post attempt #2, trying again.

### ISSUE THREAD

**Issue by DuffLevyCGY** [ArthurG/Kijiji-Repost-Headless#100]
I am not getting any type or usable error however this is the output:

Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Failed ad post attempt #2, trying again.
Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Failed ad post attempt #3, trying again.
Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Failed ad post attempt #4, trying again.
Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Image upload success on try #1
Failed ad post attempt #5, giving up.

Is there a location I can see more detail on the error?

**Comment by ArthurG** [comment]
Are you able to post ads normally from your Kijiji account?

**Comment by DuffLevyCGY** [comment]
Yes I can, oddly enough we get an email from Kijiji saying we have successfully posted the add on each attempt. ( 5 times) No add is posted.

**Comment by bulatbulat48** [comment]
same issue, categoryId: '166'

**Comment by DuffLevyCGY** [comment]
Also categoryId: '166'

**Comment by jackm** [comment]
Any category under the "Services" group requires an additional HTTP POST parameter to be sent otherwise the Kijiji site will reject it and it won't get posted.

If you post an ad manually, you can see there is an additional form selection for the chosen "package" that is required. I created a test ad manually and looked at the HTTP POST request headers and I saw one additional parameter that seems to be relating to the chosen feature package.

Try adding this to your ad yaml posting file and see if Kijiji accepts the ad (chooses the basic package, $19.95/mo):
`featuresForm.featurePackage: 'PKG_BASIC'`

**Comment by bulatbulat48** [comment]
@jackm added to file item.yml, same issue:
Image upload success on try #1
Failed ad post attempt #2, giving up.

**Comment by yasas100** [comment]
Having the same issue. Positing in phones category. It posted the ad 5 times and says 
Image upload success on try #1
Failed ad post attempt #2, trying again.
Image upload success on try #1
Failed ad post attempt #3, trying again.
Image upload success on try #1
Failed ad post attempt #4, trying again.
Image upload success on try #1

**Comment by jordanmehravar** [comment]
Can one of you attach the YML file/Images that you're using? Interestingly, I was able to post in category 166 without any issues.

**Comment by yasas100** [comment]
[item.yml.txt](https://github.com/ArthurG/Kijiji-Repost-Headless/files/2299750/item.yml.txt)

**Comment by jordanmehravar** [comment]
![image](https://user-images.githubusercontent.com/22223443/44303886-4ceedd80-a31a-11e8-8b9d-83dbcb9b9288.png)

I was able to recreate the bug using your file and my own image. However, my posts went through. I'll look more into this issue.

**Comment by jordanmehravar** [comment]
@yasas100 

I might have solved your problem. In the YML file you provided, the ad title has an extra space. Change:

**postAdForm.title: 'Iphone 7 Unlocked 32GB '**

to

**postAdForm.title: 'Iphone 7 Unlocked 32GB'**

I was able to recreate the bug by adding this extra space to any ad title. Let me know if this works.
